src/pages/edge/adobe-journey-optimizer-decisioning/api-reference.md

```diff
@@ -104,13 +104,33 @@ iOS
 
 <Tabs query="platform=ios&api=update-propositions"/>
 
+## updatePropositions
```

Comment:
Can we call this updatePropositionsWithCompletionHandler to differentiate it from the other API? Here's an example from Media SDK have we have similar set of APIs
https://developer.adobe.com/client-sdks/solution/adobe-media-analytics/api-reference/#createtracker
https://developer.adobe.com/client-sdks/solution/adobe-media-analytics/api-reference/#createtrackerwithconfig